src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsController.java

```diff
@@ -233,35 +218,38 @@ public String getProjectSpecificPage(@PathVariable Long projectId, final Model m
 	/**
 	 * Gets the name of the template for the new project page
 	 *
-	 * @param useCartSamples
-	 *            Whether or not to use the samples in the cart when creating
-	 *            the project
-	 * @param model
-	 *            {@link Model}
-	 *
+	 * @param useCartSamples Whether or not to use the samples in the cart when creating
+	 *                       the project
+	 * @param model          {@link Model}
+	 * @param owner          whether or not to lock the sample(s) from being modified from new
+	 *                       the project
 	 * @return The name of the create new project page
 	 */
 	@RequestMapping(value = "/projects/new", method = RequestMethod.GET)
 	public String getCreateProjectPage(
 			@RequestParam(name = "cart", required = false, defaultValue = "false") boolean useCartSamples,
-			final Model model) {
+			final Model model,
+			@RequestParam(name = "lockSamples", required = false, defaultValue = "true") boolean owner) {
 		model.addAttribute("useCartSamples", useCartSamples);
+		model.addAttribute("lockSamples", owner);
```

Comment:
Is this actually used in the view?  I don't see anything using `${lockSamples}` in `project_new.html`

Reply:
Fixed in 9e258b0be30f71fb335b7baf9d4591ba3e44601b